internal/resources/utils/clog/clog.go

```diff
@@ -0,0 +1,83 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package clog
```

Comment:
Personally, I'm not a fan of him this under `resources/utils`, it's not straight forward

My suggestion is to have a folder called `clog` at the root of `internal`

In my personal projects I would have this `clog` package under a folder that holds packages that are technology focused, and not domain. Some people call it `infra`. We don't have such structure, I'm sharing it just because I like it :)

Reply:
moved it to `internal/infra/clog`. good call!